### ISSUE TITLE
[Spring] --invoker-package flag not working for --lang spring

### ISSUE THREAD

**Issue by qin** [swagger-api/swagger-codegen#5178]
##### Description

--invoker-package flag is not working for --lang spring, so that the generated main class like Swagger2SpringBoot.java is always under io.swagger package.

##### Swagger-codegen version

2.2.2

##### Swagger declaration file content or url

N/A

##### Command line used for generation

swagger-codegen \
  generate \
  --input-spec swagger.yaml \
  --invoker-package com.example \
  --group-id com.example \
  --artifact-id api \
  --lang spring \
  --output ./rest

##### Steps to reproduce

1. use the command line above
2. inspect the generated code, and you can see no package of com.example is generated.

##### Related issues

N/A

##### Suggest a Fix

N/A

**Comment by wing328** [comment]
@qin thanks for reporting the issue. `invokerPackage` is not used in the spring templates. Can you please try `basePackage` instead?
```
swagger-codegen|master ⇒ java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar config-help -l spring | grep -i package
	modelPackage
	    package for generated models
	apiPackage
	    package for generated api classes
	invokerPackage
	    root package for generated code
	configPackage
	    configuration package for generated code
	basePackage
	    base package for generated code
```

We'll file a PR to remove `invokerPackage` for Java Spring generator.

**Comment by qin** [comment]
Great, thanks~

**Comment by balzaczyy** [comment]
When will this be released? 2.2.3 still has this issue.

**Comment by wing328** [comment]
Sorry there's no ETA at the moment. Please use the SNAPSHOT version mentioned in the README or build the latest master by cloning the repo.
